client/lib/products-list/index.js

```diff
@@ -45,7 +45,11 @@ ProductsList.prototype.get = function() {
 		debug( 'First time loading ProductsList, check store' );
 
 		if ( typeof localStorage !== 'undefined' ) {
-			data = localStorage.getItem( 'ProductsList' );
+			try {
+				data = JSON.parse( localStorage.getItem( 'ProductsList' ) );
```

Comment:
You could store it with `localForage` instead; you'd save the `try/catch` loop and the manual serialisation/deserialisation. It'll make thing async though, so if that's a pain for the quick fix I guess don't worry...

Reply:
I'd rather not go replaces this with localForage here, we can make a separate PR for replacing it.